### DISCUSSION TITLE
基于Higress、通义千问以及 ChatGPT-Next-Web，AI 代理能不能提供一个详细的操作部署

### DISCUSSION THREAD

**Discussion by Liyu910228** [higress-group/higress#1216] [Q&A]
按照Higress公众号上面文章部署到一半部署不下去，麻烦出一个详细操作手册。
<img width="960" alt="c862b9839fe4f76f476040a0cd77aab" src="https://github.com/user-attachments/assets/5d641160-326f-49e0-9bce-5fecc87bd892">

**Comment by johnlanni** [comment]
https://github.com/alibaba/higress/issues/938

https://github.com/alibaba/higress/issues/1023

可以看下这两个

**Comment by Liyu910228** [comment]
早就看过了，也自己搭建过了；我需要的是Higress配置页面操作，有时候也不一定用ChatGPT-Next-Web

**Comment by johnlanni** [comment]
@Liyu910228 按 #1023 的配置，然后进浏览器里访问 http://localhost:8001/ 打开 Higress 控制台，可以看到控制台的配置
